src/views/stack_view.rs

```diff
@@ -15,6 +16,7 @@ pub struct StackView {
     // Store layers from back to front.
     layers: Vec<Child>,
     last_size: Vec2,
+    dirty: cell::Cell<bool>,
```

Comment:
We could use a little comment here on what `dirty` is for.

Reply:
no problem, I just need to figure out how to check out the PR